house/models.py

```diff
@@ -46,8 +46,7 @@ class Job(models.TextChoices):
 
 
 class House(models.Model):
-    house_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    name = models.CharField(max_length=50)
```

Comment:
maybe we should leave the name for the user to Identify his houses by a given non unique name he decides on wyt?

Reply:
I agree with @shlior7 this could be a nice to have 👍

Reply:
Sure we can, i will add this field